test/src/test/java/jenkins/security/s2m/AdminFilePathFilterTest.java

```diff
@@ -40,9 +41,14 @@
 import java.io.PrintWriter;
 import java.io.StringWriter;
 import java.lang.reflect.Field;
+import java.util.List;
 import java.util.logging.Level;
 import javax.inject.Inject;
 import jenkins.SoloFilePathFilter;
+import jenkins.telemetry.impl.SlaveToMasterFileCallableUsage;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.hasSize;
```

Comment:
Spotless expects static imports on top.

Reply:
:angry:

Reply:
I know, I took care of that for you in the private PRs because we had a deadline, but here you're on your own 😛